tuned/plugins/plugin_cpu.py

```
@@ -775,12 +775,18 @@ def _set_boost(self, boost, device, instance, sim, remove):
 
+		boost = self._cmd.get_bool(boost)
 		if os.path.exists(self._pstate_boost_path(cpu_id)):
 			if not sim:
 				if boost == "0" or boost == "1":
-					self._cmd.write_to_file(self._pstate_boost_path(cpu_id), boost, \
-						no_error = [errno.ENOENT] if remove else False, ignore_same=True)
 					log.info("Setting boost value '%s' for cpu '%s'" % (boost, device))
+					# EINVAL if boost isn't supported
+					_no_error = [errno.EINVAL]
+					if remove:
+						_no_error = _no_error + [errno.ENOENT]
```

Ruff (0.15.12)

[warning] Consider `[*_no_error, errno.ENOENT]` instead of concatenation

Replace with `[*_no_error, errno.ENOENT]`

(RUF005)